src/codegen/pass2_generate.cc

```diff
@@ -919,19 +937,35 @@ static void emit_accept(
     // jump table
     if (opts->computed_gotos && nacc >= opts->computed_gotos_threshold && !have_tags) {
         CodeList* block = code_list(alc);
+        uint32_t min_index = UINT32_MAX;
```

Comment:
Please don't use a shadowed variable (the first definition on line 789), reuse the outer one.